docs/EN_US/ContainerizedHPCC/ContainerizedMods/CustomConfig.xml

```diff
@@ -992,4 +991,169 @@ eclagent:
       </sect3>
     </sect2>
   </sect1>
+
+  <sect1 id="LDAPSecurity_forConatiners">
+    <title>LDAP Security for Containers</title>
+
+    <para>Containerized LDAP support for HPCC Systems is accomplished by
```

Comment:
This is not exactly correct. What is being described here is how to make the HPCC admin credentials safe and secure. Using LDAP in a containerized deployment does not require using secrets. There should be documentation elsewhere on how to generally configure LDAP in a containerized system. That documentation should refer to this section/chapter on how to secure the HPCC admin credentials.

Reply:
Perhaps we should open a JIRA to document generally configuring LDAP for a containerized system?

Reply:
Yes,, I think it should be documented.

Reply:
created JIRA https://hpccsystems.atlassian.net/browse/HPCC-31516
please enter relevant information into that JIRA or assign to whomever has that info